Docs/developer_guide/build_instructions/windows.md

```diff
@@ -2,7 +2,7 @@
 
 ## Install prerequisites
 
-- [CMake](http://www.cmake.org/cmake/resources/software.html) version >= 3.21.1 (or 3.15.1 <= version < 3.20.4)
+- [CMake](http://www.cmake.org/cmake/resources/software.html) version that meets at least the minimum required CMake version [here](https://github.com/Slicer/Slicer/blob/master/CMakeLists.txt#L1). Avoid CMake 3.21.0 specifically.
```

Comment:
@lassoan Here is where it is updated in the build instructions. I have it pointing directly to the place in the file so it wouldn’t get out of sync. 

However thinking about how the documentation gets tagged with a release, linking to `master` here could lead to issues when the `master` branch doesn’t match the stable version doc.

I’ll probably have to change this line to a specific text version and not have this type of link.